src/content/blog/whats-new-august-2025.mdx

```diff
@@ -0,0 +1,355 @@
+- 🚀 [Astro's most popular integration](https://astro.build/integrations/) passes 1 million monthly downloads
+- 🧑‍💻 1 million installs of the [Astro VSCode extension](https://marketplace.visualstudio.com/items?itemName=astro-build.astro-vscode)
+- 👋 Summer merch will soon be disappearing from the [Astro swag shop](https://shop.astro.build/). Get it before it's gone! 
+
+
+## Latest releases
+
+Minor releases introduce new features and improvements. Check out what's new!
+
+- [Astro 5.13](https://github.com/withastro/astro/blob/refs/heads/main/packages/astro/CHANGELOG.md#5130) - experimental improvements to environment variable handling, support for Chrome DevTools workspaces, and more!
+
+## Astro Adopters
+
+From Travelodge to Renault, NixOS to Drizzle, some of the biggest names you know are using Astro!
+
+export const featuredSites = [
+  'orm.drizzle.team',
+  'nixos.org',
+  'pricefinder.travelodge.co.uk',
+  'ledefile.renault.com',
+]; 
+
+{/* Use for grids of 2 per row  */}
+<div class="grid grid-cols-1 sm:grid-cols-2 gap-5">
+  <ShowcaseCard site={await getEntry('showcase', 'pricefinder.travelodge.co.uk')} />
+  <ShowcaseCard site={await getEntry('showcase', 'ledefile.renault.com',)} />
+  <ShowcaseCard site={await getEntry('showcase', 'nixos.org')} />
+  <ShowcaseCard site={await getEntry('showcase', 'orm.drizzle.team')} />
+</div>
+
+
+{/* Can use when there are multiples of 3  */}
+{/* <ShowcaseGridById ids={featuredSites} />  */}
+
+See more companies using Astro in our [site showcase](/showcase/), and explore Astro for corporate or enterprise use with one of our [agency partners](/agencies/)!
+
+## Weird Wide Web
+
+Some sites from the last month that show off what you can do with Astro that are quirky, innovative, inspiring, or just plain fun!
+
+- Play a [geography sorting game](https://geolineup.com/) built with Astro! Test your knowledge as you drag and drop city names to put them in order from north to south.
+- Why look out the window when you can view [the current sky at your approximate location](https://sky.dlazaro.ca/), rendered as a CSS gradient? ((source code)[https://github.com/dnlzro/horizon])
+
+
+
+export const monthlySites = [
+  'geolineup.com',
+  'sky.dlazaro.ca',
+]; 
+
+{/* Use for grids of 2 per row  */}
+<div class="grid grid-cols-1 sm:grid-cols-2 gap-5">
+  <ShowcaseCard site={await getEntry('showcase', 'geolineup.com')} />
+  <ShowcaseCard site={await getEntry('showcase', 'sky.dlazaro.ca')} />
+</div>
+
+{/* Can use when there are multiples of 3  */}
+{/* <ShowcaseGridById ids={monthlySites} /> */}
+
+## Cosmic Canvas
+
+Didn't think you could do that with Astro? These stunning sites with spectacular visuals and smooth animations prove you can!
+
+export const canvasSites = [
+  'ledefile.renault.com',
+  'www.thelonelypixel.co.uk',
+]; 
+
+{/* Use for grids of 2 per row  */}
+
+<div class="grid grid-cols-1 sm:grid-cols-2 gap-5">
+  <ShowcaseCard site={await getEntry('showcase', 'ledefile.renault.com')} />
```

Comment:
Renault is also in the "Astro Adopters" section above, I guess it's fine? Just pointing it out in case it's not.